### PR TITLE
Add docs to telio-relay

### PR DESCRIPTION
**Pull request by baltrus1**
Prepare release to docs.rs
Clean up telio-relay, telio-task, telio-wg, telio-lana, device/ on the way

### :ballot_box_with_check: Definition of Done checklist
- [x] Commit history is clean ([requirements](../../blob/main/docs/git_commit_messages_requirements.md))
- [x] README.md is updated
- [x] changelog.md is updated

### CODE REVIEW

File: crates/telio-task/src/task.rs [hunk 1]
```diff
@@ -443,13 +443,13 @@ mod tests {
 
         test.stop().await;
 
-        // Task closed on drop, and droped on next context switch.
+        // Task closed on drop, and dropped on next context switch.
```
Comment (dfetti) [inline crates/telio-task/src/task.rs:446]: . at the end

File: crates/telio-task/src/task.rs [hunk 2]
```diff
@@ -175,7 +175,7 @@ where
         }
     }
 
-    /// Execute action with exlusive access on state.
+    /// Execute action with exclusive access on state.
```
Comment (dfetti) [inline crates/telio-task/src/task.rs:178]: . at the end

File: crates/telio-task/src/task.rs [hunk 3]
```diff
@@ -72,38 +72,38 @@ pub trait RuntimeExt: Runtime {
...
+///   * Wait entry to wait for outside triggers. (Like [Rx], sockets, timers etc)
 ///   * Ability to execute mutation on state without requiring mutex'es
 pub struct Task<S: Runtime> {
     stop: Arc<Notify>,
     execute: Tx<Update<S, S::Err>>,
     join: Option<JoinHandle<Result<(), S::Err>>>,
 }
 
-/// Task was stopped durring execution.
+/// Task was stopped during execution.
```
Comment (dfetti) [inline crates/telio-task/src/task.rs:98]: . at the end

File: crates/telio-task/src/task.rs [hunk 4]
```diff
@@ -54,7 +54,7 @@ pub trait Runtime: Sized {
 /// Typical responses from wait implementation.
 #[async_trait]
 pub trait RuntimeExt: Runtime {
-    /// Guard multiple async operations form being interupted by exec.
+    /// Guard multiple async operations form being interrupted by exec.
```
Comment (dfetti) [inline crates/telio-task/src/task.rs:57]: . at the end

File: crates/telio-relay/src/multiplexer/mc.rs [hunk 5]
```diff
@@ -191,15 +191,15 @@ impl Sink<(PublicKey, Packet)> for MultiChannel {
...
+            // [PollSender] flush completes after single poll;
             let _ = chan.tx.poll_flush_unpin(cx);
         }
         Poll::Ready(Ok(()))
     }
 
     fn poll_close(mut self: Pin<&mut Self>, cx: &mut Context<'_>) -> Poll<Result<(), Self::Error>> {
         for chan in self.channels.values_mut() {
-            // [PollSender] close compleates afer single poll;
+            // [PollSender] close completes after single poll;
```
Comment (dfetti) [inline crates/telio-relay/src/multiplexer/mc.rs:202]: ; at the end

File: crates/telio-relay/src/multiplexer/mc.rs [hunk 6]
```diff
@@ -191,15 +191,15 @@ impl Sink<(PublicKey, Packet)> for MultiChannel {
 
     fn poll_flush(mut self: Pin<&mut Self>, cx: &mut Context<'_>) -> Poll<Result<(), Self::Error>> {
         for chan in self.channels.values_mut() {
-            // [PollSender] flush compleates afer single poll;
+            // [PollSender] flush completes after single poll;
```
Comment (dfetti) [inline crates/telio-relay/src/multiplexer/mc.rs:194]: ; at the end

File: crates/telio-task/src/io.rs [hunk 7]
```diff
@@ -21,13 +21,13 @@ pub mod chan {
...
-    /// Default reciever channel
+    /// Default receiver channel
     pub type Rx<T> = tokio::sync::mpsc::Receiver<T>;
 }
 
-/// Channel encapsulating multi producer, single consumer send and reciever sides
+/// Channel encapsulating multi producer, single consumer send and receiver sides
 pub struct Chan<T> {
-    /// Recieve half.
+    /// Receive half.
```
Comment (dfetti) [inline crates/telio-task/src/io.rs:30]: . at the end

File: crates/telio-task/src/io.rs [hunk 8]
```diff
@@ -21,13 +21,13 @@ pub mod chan {
...
     pub rx: chan::Rx<T>,
     /// Send half.
```
Comment (dfetti) [inline crates/telio-task/src/io.rs:32]: . at the end

**Review by dfetti — DISMISSED** [review]
Looks good to me. 
I pointed some dots and semicolons (not all of them) at the end of the lines, but probably we don't care about them. You can remove them or not. All good from my side.

**Comment by baltrus1** [comment]
> Looks good to me. I pointed some dots and semicolons (not all of them) at the end of the lines, but probably we don't care about them. You can remove them or not. All good from my side.

Good point, I removed ending dots in telio-task. Probably more cleanup like this could be done in other crates, it's mostly parts I recently touched that made me do this 😄

**Review by packgron — APPROVED** [review]

**Review by packgron — APPROVED** [review]

**Pull request merged**